src/Keyborg.ts

```diff
@@ -232,8 +258,8 @@ export class Keyborg {
   private _core?: KeyborgCore;
   private _cb: KeyborgCallback[] = [];
 
-  static create(win: WindowWithKeyborg): Keyborg {
-    return new Keyborg(win);
+  static create(win: WindowWithKeyborg, props?: KeyborgProps): Keyborg {
```

Comment:
```suggestion
  static create(win: WindowWithKeyborg, props: KeyborgProps = {}): Keyborg {
```
Default options can always just be empty

Reply:
Plus it renders to a few more bytes of JS to create that object (so, it won't be cheaper than `if (props)`).